source/game/field/obj/ObjFlow.cc

```diff
@@ -0,0 +1,36 @@
+#include "game/field/obj/ObjFlow.hh"
+#include "game/system/ResourceManager.hh"
+#include <egg/util/Stream.hh>
```

Comment:
You don't use a stream here. Also there should be newlines to separate the other includes. And one before `namespace Field`
```suggestion

```